src/cljs/rems/application.cljs

```diff
@@ -639,6 +640,19 @@
                                   actions)]
                       forms)}])))
 
+(defn- render-resource [resource]
+  ^{:key (:catalogue-item/id resource)}
+  [:div.application-resource
+   (localized (:catalogue-item/title resource))
+   ;; slight duplication with rems.catalogue/catalogue-item-more-info,
+   ;; but the data has a different schema here (V2Resource vs. CatalogueItem)
+   (when-let [url (or (localized (:catalogue-item/infourl resource))
```

Comment:
Hm, this is kind of subtle, but if I have a catalogue item with EN more info link and FI link defaulting to the urn, the localized function already does the fallback to the only available language (EN) and thus the urn link does not work when the language is set to FI.

Reply:
good catch! I'll add a comment